### DISCUSSION TITLE
Sharing a printer

### DISCUSSION THREAD

**Discussion by debiantriage** [michaelrsweet/pappl#278] [Ideas]
I had an idea that I promoted [here](https://github.com/OpenPrinting/gutenprint-printer-app/discussions/2). What I had in mind was parity with a non-emulated IPP printer in terms of publishing or not on a non-local interface. I do not know whether it deserves to be classified as a bug. If it should, I do not know how to move it elsewhere.

Certainly, I do feel a user should have the ability not to share a printer, but how possible it is is a different matter in the context of PAPPL.

**Comment by michaelrsweet** [comment]
From the standpoint of PAPPL-based printer applications, I think we will be limiting the configurability to local vs. remote/shared for all printers supported by the application. Normally this is done using the "listen-hostname" option, which is supported by the [LPrint](https://github.com/michaelsweet/lprint) and [HP Printer App](https://github.com/michaelsweet/hp-printer-app) snaps. For other platforms things are less clean...
